From: Jan Engelhardt <ej@inai.de>
Date: 2024-11-05 01:19:04.918343561 +0100

graph.c:83:18: error: implicit declaration of function ‘strncmp’ [-Wimplicit-function-declaration]

---
 src/graph.c |    1 +
 1 file changed, 1 insertion(+)

Index: conauto-2.03/src/graph.c
===================================================================
--- conauto-2.03.orig/src/graph.c
+++ conauto-2.03/src/graph.c
@@ -18,6 +18,7 @@
 
 #include <stdlib.h>
 #include <inttypes.h>
+#include <string.h>
 
 #include "graph.h"
 
